test/extended/default_forcePull_test.go

```diff
@@ -0,0 +1,262 @@
+// +build default
+
+package extended
+
+import (
+	"fmt"
+	"path/filepath"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	buildapi "github.com/openshift/origin/pkg/build/api"
+	exutil "github.com/openshift/origin/test/extended/util"
+	"time"
+)
+
+var (
+	fpTrueS2I         = filepath.Join("fixtures", "forcepull-true-s2i.json")
+	fpFalseS2I        = filepath.Join("fixtures", "forcepull-false-s2i.json")
+	fpTrueDock        = filepath.Join("fixtures", "forcepull-true-dock.json")
+	fpFalseDock       = filepath.Join("fixtures", "forcepull-false-dock.json")
+	fpTrueCust        = filepath.Join("fixtures", "forcepull-true-cust.json")
+	fpFalseCust       = filepath.Join("fixtures", "forcepull-false-cust.json")
+	s2iDockBldr       = "docker.io/openshift/ruby-20-centos7"
+	custBldr          = "docker.io/openshift/origin-custom-docker-builder"
+	s2iStartingHexID  string
+	custStartingHexID string
+)
+
+/*
+If docker.io is not responding to requests in a timely manner, this test suite will be adversely affected.
+
+If you suspect such a situation, attempt pulling some openshift images other than ruby-20-centos7 or origin-custom-docker-builder
+while this test is running and compare results.  Restarting your docker daemon, assuming you can ping docker.io quickly, could
+be a quick fix.
+*/
+
+func corruptImage(corruptee, corruptor, strategy string) {
+	By(fmt.Sprintf("\n%s****BEFORE EACH FORCE PULL*****:  Calling docker tag to corrupt %s builder image %s by tagging %s", time.Now().Format(time.RFC850), strategy, corruptee, corruptor))
```

Comment:
also I think all the func(s) from this file really belongs to some 'helpers' packages where they can be re-used by other tests from other groups. How about moving them to `test/extended/util/image_helpers.go` ?